django/utils/http.py

```diff
@@ -33,6 +33,12 @@
 RFC3986_GENDELIMS = str(":/?#[]@")
 RFC3986_SUBDELIMS = str("!$&'()*+,;=")
 
+PROTOCOL_TO_PORT = {
+    'http': 80,
+    'https': 443,
+    'ftp': 21,
```

Comment:
Yes, `socket.getservbyname()` works on Python 2.7.9 and 3.4.2 on Linux and at least 2.6.6 on Windows.